VehNum: 1

# Direct graph matrix
DGMat: 1

# Gain matrix
GMat: 1

# Formation distances matrix
FormX: 0
FormY: 0

UseRepel: True
# Force repeling controller gain
kf: 3.5
# Force correction factor
kd: 0.0
# Force vector corr factor
ni: 0.5
# repel radius
rf: 1.8

# use external DP controller
UseExtCon: False
UseImedStart: False
kdp: 0.5

# max speed
MaxSpeed: 1.0

# Velocity gain
gamma: 0.1

# Sampling time for yaw orientation rate 
Ts: 0.7
